### PR TITLE
Anomaly #41246 : Refactor to comply code convention on wip-6.2-accounting

### PR DESCRIPTION
**Pull request by kra-axelor**

### CODE REVIEW

File: axelor-account/src/main/java/com/axelor/apps/account/web/MoveController.java [hunk 1]
```diff
@@ -203,15 +216,16 @@ public void deleteMove(ActionRequest request, ActionResponse response) throws Ax
   }
 
   protected void removeOneMove(Move move, ActionResponse response) throws Exception {
+    MoveRemoveService moveRemoveService = Beans.get(MoveRemoveService.class);
```
Comment (kra-axelor) [inline axelor-account/src/main/java/com/axelor/apps/account/web/MoveController.java:219]: Not sure about the changes I've done in this method but it seems weird to me to call `Beans.get(MoveService.class).getMoveRemoveService()` to get a instance of MoveRemoveService when you can just call `Beans.get(MoveRemoveService.class)`

Comment (ale-axelor) [reply axelor-account/src/main/java/com/axelor/apps/account/web/MoveController.java:219]: I agree, I think this is simply legacy code

File: axelor-account/src/main/java/com/axelor/apps/account/web/MoveLineController.java [hunk 2]
```diff
@@ -312,8 +314,11 @@ public void loadAccountInformation(ActionRequest request, ActionResponse respons
 
         TaxLine taxLine =
             Beans.get(MoveService.class).getTaxLine(move, moveLine, accountingAccount);
-        if (taxLine != null) response.setValue("taxLine", taxLine);
-        else response.setValue("taxLine", null);
+        if (taxLine != null) {
+          response.setValue("taxLine", taxLine);
+        } else {
+          response.setValue("taxLine", null);
+        }
```
Comment (ale-axelor) [inline axelor-account/src/main/java/com/axelor/apps/account/web/MoveLineController.java:321]: only `response.setValue("taxLine", taxLine);` seems enough

**Review by ale-axelor — CHANGES_REQUESTED** [review]

File: axelor-account/src/main/java/com/axelor/apps/account/web/MoveLineController.java (hunk 2 above)
Comment (kra-axelor) [reply axelor-account/src/main/java/com/axelor/apps/account/web/MoveLineController.java]: done

**Pull request merged**